### PR TITLE
ffmpeg: add v7.1

### PR DESCRIPTION
**Pull request by albestro**

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Verified Checksums. Looks good to me. Waiting on final approval from @xjrc.

**Comment by albestro** [comment]
@xjrc can we just proceed with this? I just added the version and it built for me. I haven't thoroughly checked if there is any new dependency requirement, in case just let me know.

Otherwise I would just go for merging this so that this new version of ffmpeg is known to spack and in case of any problem in the future we can always update/fix it.

**Review by alalazo — APPROVED** [review]
Let's follow with a new PR, in case there are issues

**Pull request merged**
